src/styles/styles.js

```diff
@@ -1,179 +1,23 @@
-// We place items a percentage to the safe area on the top or bottom of the screen
 import fontFamily from './fontFamily';
 import italic from './italic';
 import addOutlineWidth from './addOutlineWidth';
 import themeColors from './themes/default';
 import fontWeightBold from './fontWeight/bold';
 import variables from './variables';
 import colors from './colors';
+import spacing from './utilities/spacing';
```

Comment:
When I started on this review I was going to propose doing this, and you already did it! Thank you!